admin/app/controllers/authenticated/sessions/list.js

```diff
@@ -12,14 +13,27 @@ export default class SessionListController extends Controller {
   @tracked pageNumber = DEFAULT_PAGE_NUMBER;
   @tracked pageSize = 10;
   @tracked id = null;
+  @tracked status = null;
 
+  sessionStatusAndLabels = [
+    { status: '', label: 'Indifférent' },
+    { status: CREATED, label: statusToDisplayName[CREATED] },
+    { status: FINALIZED, label: statusToDisplayName[FINALIZED] },
+    { status: IN_PROCESS, label: statusToDisplayName[IN_PROCESS] },
+    { status: PROCESSED, label: statusToDisplayName[PROCESSED] },
+  ];
```

Comment:
Une petite proposition de refactoring en partant du principe que dans `statusToDisplayName` il n'y a que les status qui nous intéresse 😄 

```suggestion
  sessionStatusAndLabels = [
    { status: '', label: 'Indifférent' },
    ..._.map(statusToDisplayName, (label, status) => ({ status, label })),
  ];
```

Reply:
Ok bonne idée !